sdk/storage/Azure.Storage.Blobs/src/QuickQueryExtensions.cs

```diff
@@ -52,9 +52,18 @@ internal static class QuickQueryExtensions
                     RecordSeparator = jsonTextConfiguration.RecordSeparator?.ToString(CultureInfo.InvariantCulture)
                 };
             }
+            else if (textConfiguration.GetType() == typeof(BlobQueryArrowOptions))
```

Comment:
Do you have a separate input and output converter?

Reply:
Fixed!